README.md

```diff
@@ -31,6 +31,10 @@ When `yes`, this variable includes and execute
 during the play, which makes it possible to manage certificates without ugly
 hacks. This is only supported in `ansible` version _at least_ 2.2 and later.
 
+`reallyenglish.x509` must be listed in `requirements.yml`, or available for
```

Comment:
i think the usage is not so clear and the phrase should be written with a clear explanation.